packages/core/core/src/TargetDescriptor.schema.js

```diff
@@ -1,7 +1,7 @@
 // @flow strict-local
 import type {SchemaEntity} from '@parcel/utils';
 
-export const engines: SchemaEntity = {
+export const ENGINES_SCHEMA: SchemaEntity = {
```

Comment:
would be nice to store these schemas as json, but we'd lose the flow typing of this literal 😞

Reply:
Exactly...